README.md

```diff
@@ -1,92 +1,40 @@
-# waterthetrees
+# Water the Trees - Frontend
 
-Water The Trees is a platform that crowd sources tree planting and maintenance. We believe in the power of trees to restore natural habitat for animals, insects, and fauna. We are interesting in continuous massive tree planting events to help sequester carbon and stabilize climate change extremes.
+Water the Trees (WTT) is a platform for crowd-sourced tree-planting and maintenance. We believe in the power of trees to restore natural habitats for wildlife and sequester carbon to stabilize the effects of climate change. 
 
-## Join Us!
+WTT is an open-source project run by volunteers at [Sierra Club San Francisco Bay Chapter](https://www.sierraclub.org/san-francisco-bay), [San Francisco Public Works](https://www.sfpublicworks.org/), and [Code for San Francisco](https://www.sfpublicworks.org/), part of the [Code for America](https://www.codeforamerica.org/) Brigade Network.
 
-https://docs.google.com/document/d/1L5Hc8_K_NhVhAejdE05C_Y__CgqeBWFrFYFoNqBSBbQ/edit
-Water the Trees is an open source project run by tree planting volunteers at Sierra Club, Public Works, and developers at Code for America.
+## Want to contribute?
 
-### INSTALL GIT
+First, check out the [WTT Onboarding Guide](https://docs.google.com/document/d/1L5Hc8_K_NhVhAejdE05C_Y__CgqeBWFrFYFoNqBSBbQ/edit?usp=sharing). Then, follow the instructions below.
 
-1. install git https://git-scm.com/downloads
+### Setup
 
-### INSTALL DOCKER
-
-1. install docker https://www.docker.com/get-started
-2. install docker-compose (only on linux, on osx this installs with docker)
-
-### CLONE REPO
-
-1. Open terminal and run this
+New to coding? Before proceeding, be sure to install [Xcode command line tools](https://www.freecodecamp.org/news/install-xcode-command-line-tools/), [Node](https://nodejs.org/en/download/), [Yarn](https://classic.yarnpkg.com/en/docs/install), and [Git](https://git-scm.com/book/en/v2/Getting-Started-Installing-Git).
```

Comment:
Are these tasks covered in the onboarding doc?  If so, might not need them here.  Also, Xcode and yarn aren't required (I'm not running them.)